pico/tests/resources-pico/src/main/java/io/helidon/pico/tests/pico/tbox/AbstractSaw.java

```diff
@@ -66,7 +66,7 @@ void setBladeList(List<AbstractBlade> blades) {
     }
 
     @Inject
-    void setBladeProviders(List<Provider<AbstractBlade>> blades) {
+    public void setBladeProviders(List<Provider<AbstractBlade>> blades) {
```

Comment:
Why is there a change in visibility?

Reply:
I have other package privates in the abstract base - i needed one that was public for testing.